test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -85,8 +85,14 @@ public void testLoadResponseGeneratorModeOfOpNoSuchFactoryMethod() {
     for (Map.Entry<String, String> entry : configEntries.entrySet()) {
       TestHelper.setConfigValue(config, entry.getKey(), entry.getValue());
     }
-    thrown.expect(InvalidConfigurationException.class);
-    DatabaseAdaptor.loadResponseGenerator(config);
+    try {
+      DatabaseAdaptor.loadResponseGenerator(config);
+      fail("Expected an InvalidConfigurationException");
+    } catch (InvalidConfigurationException ice) {
+       if (!ice.getMessage().contains("noThisMethod cannot be parsed as a fully qualified method")) {
```

Comment:
Sadly, the error message actually says "quailfied". I think we should fix the typo in DatabaseAdaptor rather than expecting it here. Also, it says "name" and not "method" at the end, which we should fix here. ;-)